src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -25,94 +53,1321 @@
     public String groupByTeam = null;
     public String showStats = null;
     public int startIndex;
+    private boolean isPanelsCollapsed;
+    
+    private FieldValidator validator = new FieldValidator();
 
     // used for html table ajax loading
     public String courseId = null;
     public String feedbackSessionName = null;
     public String ajaxStatus = null;
     public String sessionResultsHtmlTableAsString = null;
+    
 
+    // for question view
+    List<InstructorResultsQuestionTable> questionPanels;
+    // for giver > question > recipient, recipient > question > giver...
+    LinkedHashMap<String, InstructorFeedbackResultsSectionPanel> sectionPanels;
+    
+    // TODO multiple page data classes inheriting this for each view type, 
+    // rather than an enum determining behaviour in many methods
+    ViewType viewType;
+    enum ViewType {
+        QUESTION, GIVER_QUESTION_RECIPIENT, RECIPIENT_QUESTION_GIVER;
+        
+        public String toString() {
+            return name().toLowerCase().replaceAll("_", "-");
+        }
```

Comment:
Should be good to have a comment explaining why or where the `toString()` is used to require that modification.